helm-chart/ray-cluster/values.yaml

```diff
@@ -16,6 +16,11 @@ fullnameOverride: ""
 imagePullSecrets: []
   # - name: an-existing-secret
 
+common:
```

Comment:
Can you add some comments about this? Users may not be able to understand that without comments.

Reply:
Added!